content/en/guides/core/registry/configure_registry.md

```diff
@@ -61,37 +61,48 @@ There are two different types of roles in W&B: [Team roles]({{< ref "/guides/mod
-| View a collection’s details                                    | Read             |   X    |   X    |   X   |
-| View a linked artifact’s details                               | Read             |   X    |   X    |   X   |
-| Usage: Consume an artifact in a registry with use_artifact     | Read             |   X    |   X    |   X   |
-| Download a linked artifact                                     | Read             |   X    |   X    |   X   |
-| Download files from an artifact’s file viewer                  | Read             |   X    |   X    |   X   |
-| Search a registry                                              | Read             |   X    |   X    |   X   |
-| View a registry’s settings and user list                       | Read             |   X    |   X    |   X   |
-| Create a new automation for a collection                       | Create           |        |   X    |   X   |
-| Turn on Slack notifications for new version being added        | Create           |        |   X    |   X   |
-| Create a new collection                                        | Create           |        |   X    |   X   |
-| Create a new custom registry                                   | Create           |        |   X    |   X   |
-| Edit collection card (description)                             | Update           |        |   X    |   X   |
-| Edit linked artifact description                               | Update           |        |   X    |   X   |
-| Add or delete a collection’s tag                               | Update           |        |   X    |   X   |
-| Add or delete an alias from a linked artifact                  | Update           |        |   X    |   X   |
-| Link a new artifact                                            | Update           |        |   X    |   X   |
-| Edit allowed types list for a registry                         | Update           |        |   X    |   X   |
-| Edit custom registry name                                      | Update           |        |   X    |   X   |
-| Delete a collection                                            | Delete           |        |   X    |   X   |
-| Delete an automation                                           | Delete           |        |   X    |   X   |
-| Unlink an artifact from a registry                             | Delete           |        |   X    |   X   |
-| Edit accepted artifact types for a registry                    | Admin            |        |        |   X   |
-| Change registry visibility (Organization or Restricted)        | Admin            |        |        |   X   |
-| Add users to a registry                                        | Admin            |        |        |   X   |
-| Assign or change a user's role in a registry                   | Admin            |        |        |   X   |
+| Permission                                                     | Permission Group | Viewer | Restricted Viewer | Member | Admin | 
```

Comment:
This table is confusing. Use a checkmark to indicate Yes, and an X to indicate No.